packages/fast-usdc-contract/src/exos/status-manager.ts

```diff
@@ -82,16 +59,37 @@ export const prepareStatusManager = (
   { marshaller, routeHealth }: StatusManagerPowers,
 ) => {
   /**
-   * Keyed by a tuple of the Noble Forwarding Account and amount.
+   * Keyed by Noble Forwarding Account
+   *
+   * Transactions that are `Advanced`, `AdvanceFailed`, or `AdvanceSkipped`
+   * and are awaiting a mint.
    */
-  const pendingSettleTxs: MapStore<PendingTxKey, PendingTx[]> = zone.mapStore(
+  const pendingSettleTxs: MapStore<NobleAddress, PendingTx[]> = zone.mapStore(
```

Comment:
I see that until `migratePendingSettleStore` the key is still `PendingTxKey` but statically the code should assume `NobleAddress`.